src/rgw/rgw_lc.cc

```diff
@@ -1671,15 +1672,15 @@ int RGWLC::process(LCWorker* worker, bool once = false)
   return 0;
 }
 
-bool RGWLC::expired_session(time_t started)
+bool RGWLC::expired_session(const DoutPrefixProvider *dpp, time_t started)
```

Comment:
Same here, the dpp here is not needed, it should be a `this` since `RGWLC` is a `DoutPrefixProvider`.